client/src/components/SelectedFiltersBar.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Button, Dropdown, Grid, Header, Icon } from 'semantic-ui-react';
+import { array, func, number } from 'prop-types';
+
+const SORT_BY_OPTIONS = [
+  {
+    text: 'Deadline',
+    value: 'Deadline',
+  },
+];
+
+const PER_PAGE_OPTIONS = [
+  {
+    text: '25',
+    value: 25,
+  },
+  {
+    text: '50',
+    value: 50,
+  },
+  {
+    text: '100',
+    value: 100,
+  },
+];
+
+const propTypes = {
+  selectedCategories: array.isRequired,
+  selectedIndustries: array.isRequired,
+  minSalary: number.isRequired,
+  handleRemoveCategory: func.isRequired,
+  handleRemoveIndustry: func.isRequired,
+  handleRemoveMinSalary: func.isRequired,
+  handleClearFilters: func.isRequired,
+};
+
+const SelectedFiltersBar = ({
+  selectedCategories,
+  selectedIndustries,
+  minSalary,
+  handleRemoveCategory,
+  handleRemoveIndustry,
+  handleRemoveMinSalary,
+  handleClearFilters,
+}) => (
+  <Grid.Row>
+    <Grid.Column width={3} />
```

Comment:
Anybody know a better way to deal with positioning? I'm new to semantic-ui (and don't have a lot of FE experience in general). I just used the grid with columns of different widths.

Reply:
I haven't done much with semantic-ui either... @stephqwu ?

Reply:
That should work.. I've been thinking about it but haven't recalled a better away